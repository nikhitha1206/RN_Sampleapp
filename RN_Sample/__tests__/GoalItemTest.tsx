import { render, fireEvent, cleanup } from "@testing-library/react-native";
import React from 'react';
import { Button } from "react-native";
import renderer from 'react-test-renderer';
import GoalItem from "../components/GoalItem";
import { configure ,shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
afterEach(cleanup);
describe("render",()=>{
    const onEventMock = jest.fn();
    test('renders correctly', () => {
        const tree = renderer.create(<GoalItem onDelete={onEventMock}/>).toJSON();
        expect(tree).toMatchSnapshot();
      });  
      
})