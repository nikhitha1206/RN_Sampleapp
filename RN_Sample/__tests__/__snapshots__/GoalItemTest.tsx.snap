// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#ccc",
        "borderColor": "black",
        "borderWidth": 1,
        "marginVertical": 10,
        "padding": 10,
      }
    }
  >
    <Text />
  </View>
</View>
`;
